pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "MarkupSafe>=2.1",
     "minio>=7.0.0,!=7.2.1",
     "nbclient>=0.5.1",
-    "nbconvert>=6.5.1",
+    "nbconvert==7.1.0",
```

Comment:
fine, yes.
Should also be added to https://github.com/elyra-ai/elyra/blob/main/etc/generic/requirements-elyra.txt#L11

Reply:
Please also add there

Reply:
@harshad16 I basically said, it absolutely makes sense to update the version of nbconvert in two places simultaneously.